src/main/kotlin/com/workos/organizations/OrganizationsApi.kt

```diff
@@ -14,12 +14,39 @@ import com.workos.organizations.models.OrganizationList
  * The OrganizationsApi provides convenience methods for working with WorkOS Organizations.
  */
 class OrganizationsApi(private val workos: WorkOS) {
+  /**
+   * Options when setting the domains for an organization.
+   *
+   * @param domain The domain of the organization.
+   * @param state The verificaction state of the domain.
+   */
+  class OrganizationDomainDataOptions @JvmOverloads constructor(
+    val domain: String,
+
+    val state: OrganizationDomainDataState
+  )
+
+  /**
+   * The verification state of the domain
+   *
+   * @property PENDING The domain is pending verification.
+   * @property VERIFIED The domain has been verified. Pass this if you've already verified the domain through other methods.
+   */
+  enum class OrganizationDomainDataState {
+    @JsonProperty("verified")
+    VERIFIED,
+
+    @JsonProperty("pending")
+    PENDING
+  }
+
   /**
    * Parameters for [createOrganization].
    * Use `CreateOrganizationOptions.builder()` to create a new builder instance.
    *
    * @param name The name of the organization.
    * @param allowProfilesOutsideOrganization Whether the Connections within this Organization should allow Profiles that do not have a domain that is present in the set of the Organization's User Email Domains.
+   * @param domainData A list of data for the domains of the organization.
```

Comment:
Should we use `@deprecated` here for the documentation too? https://docs.oracle.com/javase/1.5.0/docs/guide/javadoc/deprecation/deprecation.html

Reply:
Oh, are these Javadoc comments or KDoc?

https://kotlinlang.org/docs/kotlin-doc.html#suppress

> KDoc does not support the @deprecated tag. Instead, please use the @Deprecated annotation.

Reply:
Yes, that's right. Good to know about this.